text/000x-assethub.md

```diff
@@ -0,0 +1,92 @@
+| itemDeposit               | 0.001 KSM                        | 0.023 $                | 0.5 $                         | 0.0001 KSM                | 0.0023 $                         | 0.05 $                                 |
+| metadataDepositBase       | 0.006709666617 KSM               | 0.15432183319 $        | 3.3548333085 $                | 0.0006709666617 KSM       | 0.015432183319 $                 | 0.33548333085 $                        |
+| attributeDepositBase      | 0.00666666666 KSM                | 0.15333333318 $        | 3.333333333 $                 | 0.000666666666 KSM        | 0.015333333318 $                 | 0.3333333333 $                         |
+
+```
+> Note: This is only a proposal for change and it can be modified upon additional conversation.
+
+## Drawbacks
+The contemplation of modifying deposit requirements necessitates a balanced assessment of the potential drawbacks. Highlighted below are cogent points extracted from the discourse on the [Polkadot Forum conversation](https://forum.polkadot.network/t/polkadot-assethub-high-nft-collection-deposit/4262), which provide critical perspectives on the implications of such changes:
+
+> But NFT deposits were chosen somewhat arbitrarily at genesis and it’s a good exercise to re-evaluate them and adapt if they are causing pain and if lowering them has little or no negative side effect (or if the trade-off is worth it).
+>  -> joepetrowski
+
+> Underestimates mean that state grows faster, although not unbounded - effectively an economic subsidy on activity. Overestimates mean that the state grows slower - effectively an economic depressant on activity.
+>  -> rphmeier
+
+> Technical: We want to prevent state bloat, therefore using state should have a cost associated with it.
+>  -> joepetrowski
+
+
+## Testing, Security, and Privacy
+
+## Performance, Ergonomics, and Compatibility
+
+### Performance
+This change is not expected to have a significant impact on the overall performance of the Polkadot Asset Hub. However, it is crucial to monitor the network closely, especially in the initial stages after implementation, to identify and mitigate any potential issues.
+ 
+Additionally, a supplementary proposal aims to augment the network's adaptability:
+
+> Just from a technical perspective; I think the best we can do is to use a weak governance origin that is controlled by some consortium (ie. System Collective).
+> This origin could then update the NFT deposits any time the market conditions warrant it - obviously while honoring the storage deposit requirements.
+> To implement this, we need RFC#12 and the Parameters pallet from @xlc.
+>  -> OliverTY
+
+This dynamic governance approach would facilitate a responsive and agile economic model for deposit management, ensuring that the network remains both accessible and robust in the face of market volatility.
```

Comment:
I don't really agree with this approach for a few reasons. One, in attempting to provide stability it actually removes it because the DOT-based deposit can change so often that NFT issuers don't even know how much DOT they should acquire in order to plan a few weeks in advance.

Two, it does not scale. There are deposits all over the system. This group could then find themselves perpetually changing tons of deposits.

I don't see any need to update deposits more than every six months, which is well within the realm of runtime upgrades. If deposits should be tied to some fiat value, then on Asset Hub they should just directly declare the deposit in a chosen stablecoin. If the deposits are in DOT, and users want to hold stablecoins, then wallets can easily batch an Asset Conversion swap in front of a creation call.